src/presentation/ui/layout/sidebar.templ

```diff
@@ -0,0 +1,125 @@
+package layout
+
+import infraEnvs "github.com/speedianet/os/src/infra/envs"
+
+type SidebarItem struct {
+	Label             string
+	Icon              string
+	Link              string
+	AdditionalClasses string
+}
+
+func sidebarItemsGetter(currentUrl string) []SidebarItem {
+	sidebarItems := []SidebarItem{
+		{Label: "Overview", Icon: "ph-speedometer", Link: "/_/#/overview/"},
+		{Label: "Accounts", Icon: "ph-users-three", Link: "/_/#/accounts/"},
+		{Label: "Databases", Icon: "ph-database", Link: "/_/#/databases/"},
+		{Label: "Runtime", Icon: "ph-code", Link: "/_/#/runtime/"},
+		{Label: "Cron Jobs", Icon: "ph-clock", Link: "/_/#/crons/"},
+		{Label: "File Manager", Icon: "ph-files", Link: "/_/#/file-manager/"},
+		{Label: "Mappings", Icon: "ph-graph", Link: "/mappings/"},
+		{Label: "SSL Certificate", Icon: "ph-lock", Link: "/_/#/ssl/"},
+		{Label: "Marketplace", Icon: "ph-storefront", Link: "/_/#/marketplace/"},
+		{Label: "Apps", Icon: "ph-circles-four", Link: ""},
+		{Label: "Domains", Icon: "ph-globe", Link: ""},
+		{Label: "Backups", Icon: "ph-cloud-arrow-up", Link: ""},
+		{Label: "Web Servers", Icon: "ph-browsers", Link: ""},
+		{Label: "Metrics", Icon: "ph-chart-bar", Link: ""},
+		{Label: "Logs", Icon: "ph-log", Link: ""},
+		{Label: "Image Optimizer", Icon: "ph-image-square", Link: ""},
+		{Label: "Security", Icon: "ph-shield-checkered", Link: ""},
+		{Label: "Terminal", Icon: "ph-terminal-window", Link: ""},
+	}
+
+	for itemIndex, item := range sidebarItems {
+		if item.Link != currentUrl {
+			continue
+		}
+
+		sidebarItems[itemIndex].AdditionalClasses = " text-speedia-500"
+		break
+	}
+
+	return sidebarItems
+}
+
+templ Sidebar(currentUrl string) {
+	<!-- Sidebar JavaScript -->
+	<script defer type="text/javascript">
+	function clearSession() {
+		document.cookie = "os-access-token=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;";
+		window.location.href = "/_/#/login";
+	}
+	function openSettings() {
+		window.location.href = "/_/#/settings";
+	}
```

Comment:
We are not having a settings on the new front, you may remove this. Settings will be a totally different thing soon.

Reply:
Done!